src/Angor/Client/Pages/Wallet.razor

```diff
@@ -1888,17 +1866,25 @@ else
         verificationPassed = false;
         showVerificationError = false;
     }
-    
+
     private void SkipVerification()
     {
         verificationPassed = true;
         showVerificationError = false;
         NextStep();
     }
-    
+
     protected override void OnInitialized()
     {
         NavMenuState.SetActivePage("wallet");
         base.OnInitialized();
     }
+
+    private long GetRecommendedMinFee()
```

Comment:
what will this do exactly? do we need it?

Reply:
This method has existed previously and is responsible for returning a recommended minimum fee rate based on current network conditions.

Reply:
ah so this is basically taking the result form the api call and presenting a default fee of the network right now.
yeah ok we defo need this